inyoka_theme_default/templates/planet/index.html

```diff
@@ -77,15 +82,24 @@
                     <div class="text">{{ article.text }}</div>
                   </div>
                 </div>
-                <div class="planet-article-footer">
-                  <a href="{{ article|url|e }}">{% trans %}Permanent link{% endtrans %}</a>
+
+                <ul class="planet-article-footer">
```

Comment:
1. to use `.list-inline`
2. it saves markup if the styles are directly applied to the `ul.planet-article-footer` instead of a div-wrapper
3. just look at the pure HTML in w2m etc.: Which markup tells more? (rhetoric question…)